src/containers/BooksList.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Book from '../components/Book';
+
+
+const BooksList = () => {
+  const library = useSelector(state => state.books);
```

Comment:
[Optional]
I guess you're making use of Hooks to get the state. I am curious, this is also fetching books from the store?

To avoid future problem, cause you will be told to be make use of it in the next milestone, let get familiar with it, you can make use of `mapStateToProps` to fetch books from the store

You could have something like this 👇 
```
const mapStateToProps = state => ({
  books: state.books,
});
```